Makefile

```diff
@@ -188,6 +188,15 @@ githooks-commit:
 githooks-push:
 	@if [ ! -e .git/hooks/pre-push ]; then ln -s ../../bin/pre-push .git/hooks/pre-push; fi
 
+# generate a new shrinkwrap
+shrinkwrap:
+	@rm -rf node_modules
+	@rm npm-shrinkwrap.json
```

Comment:
If `npm-shrinkwrap.json` doesn't exists for some reason, this produces an error.